godot_rl/wrappers/stable_baselines_wrapper_single_obs_space.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+import gymnasium as gym
+import numpy as np
+from godot_rl.core.godot_env import GodotEnv
+from godot_rl.core.utils import can_import, lod_to_dol
+from stable_baselines3 import PPO
+from godot_rl.wrappers.stable_baselines_wrapper import StableBaselinesGodotEnv
+from stable_baselines3.common.vec_env.base_vec_env import VecEnv
+from stable_baselines3.common.vec_env.vec_monitor import VecMonitor
+
+# A variant of the wrapper that only supports a single obs space from the dictionary ("obs")
+# this allows some basic support for using envs that have a single obs space with policies other than MultiInputPolicy
+
+class StableBaselinesGodotEnvSingleObsSpace(StableBaselinesGodotEnv):
```

Comment:
This class name and the file name are getting a bit long

Reply:
Agreed, changed.